app/uk/gov/hmrc/helptosavestub/controllers/UserInfoAPIController.scala

```diff
@@ -68,8 +69,11 @@ class UserInfoAPIController extends BaseController {
         val message = "Could not find Authorization in header"
         Logger.error(message)
         Unauthorized(message)
-      } { token ⇒
-        userInfoGen.seeded(token).fold(
+      }{ token ⇒
+        val headerToken = headers.get("token")
```

Comment:
OK but I thought a token was placed in the headers as part of the request coming from auth-login-stub, the token being entered in by the user